src/app/shared/form/advanced-confirm.directive.ts

```diff
@@ -0,0 +1,40 @@
+import { Directive, Input, HostListener, ElementRef } from '@angular/core';
+import { ModalService } from '../../core';
+
+export const AdvancedConfirmText = {
+  newFeedUrl: function (changed, originalValue, changedValue) {
+    if (changed) {
+      let prompt = "Are you sure you want to change New Feed URL";
+      if (originalValue) {
+        prompt += ` from "${originalValue}"`;
+      }
+      if (changedValue) {
+        prompt += ` to "${changedValue}"`;
+      }
+      prompt += '? This will point your subscribers to a new feed location.';
```

Comment:
If you did just pass in the entire string via the `[confirm]="someConfirmationMessage"`, this directive could be much more field-agnostic.